### DISCUSSION TITLE
Amazon Marketing Stream for sponsored display performance

### DISCUSSION THREAD

**Discussion by Lin1031** [amzn/ads-advanced-tools-docs#187] [Q&A]
Hi. We found that some targeting_texts named "placeholder keyword" show up in sd-traffic conversion data without the corresponding records in reporting version 2, target report of Sponsored Display. When you identify the cause, please let us know, thank you

**Comment by elizafarley** [comment]
Hi @Lin1031, This sounds like a bug. If this is still an issue, I'd recommend opening a [Jira ticket](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2) so the engineering team can investigate.
